pwiz_tools/Skyline/Model/Lib/BiblioSpecLite.cs

```diff
@@ -311,38 +311,59 @@ public IEnumerable<SpectrumSourceFileDetails> GetDataFileDetails()
                         // Each of these numbers is subdivided by score type.
                         // Also, select "ssf.*" because not all tables have a column "cutoffScore".

```

Comment:
Is this comment no longer accurate? If it is still is, won't the current change break whatever files don't have cutoffScore? Instead of the weird * though we can do a preliminary "HasColumn" to update the query logic. There's already a couple utility functions like that:
```
        private bool HasFileIdColumn(SQLiteConnection connection)
        {
            return SqliteOperations.ColumnExists(connection, @"RefSpectra", @"fileID");
        }
```